File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = false;
        int p_count = 0, y_count = 0;
        
        s= s.toLowerCase();
        
        for(int i = 0 ;i< s.length(); i++) {
            if(s.charAt(i) == 'p') {
                p_count++;
            }
            else if(s.charAt(i) == 'y') {
                y_count++;
            }
        }
       if (p_count == y_count){
           answer = true;
       }
        return answer;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    s= s.toUpperCase();
    let pCount = 0, yCount = 0;
    for(let i of s){
        if(i ==="P"){
            pCount++;
        }if(i ==="Y"){
            yCount++;
        }
    }
  

    return pCount === yCount ? true: false;
}